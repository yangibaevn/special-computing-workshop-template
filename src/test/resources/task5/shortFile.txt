foo bar foo
bar baz
gas