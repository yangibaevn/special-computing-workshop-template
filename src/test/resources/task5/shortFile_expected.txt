bar 2
foo 2
gas 1
baz 1